corehq/apps/domain/forms.py

```diff
@@ -1650,6 +1667,24 @@ def save(self, commit=True):
             )
             return False
 
+    def is_downgrade(self):
```

Comment:
This is rather accounting-specific logic, should it live somewhere like an `is_upgrade(oldPlan, newPlan)` function in `accounting.utils` instead? Same comment for `is_upgrade` in `domain.views`.